network/network.go

```diff
@@ -42,4 +44,18 @@ const (
 	// with this header will not be passed to the user container or
 	// included in request metrics.
 	ProbeHeaderName = "K-Network-Probe"
+
+	// Since K8s 1.8, prober requests have
```

Comment:
Do we need the version comment? :) 1.8 is so far in the past

Reply:
This is copy/pasta from Serving.  I don't think it's harming anyone to know the history 🤓

Reply:
It probably made sense when there were two constants, but now there's just one :)
🤷